Filters/src/ParticleCodeFilter_module.cc

```diff
@@ -0,0 +1,86 @@
+//
+// Simple module to select events with SimParticles with specific codes
+//
+//  Original author: Dave Brown (LBNL)
+//
+#include "art/Framework/Core/EDFilter.h"
+#include "art/Framework/Principal/Event.h"
+#include "fhiclcpp/types/Tuple.h"
+#include "fhiclcpp/types/Table.h"
+#include "fhiclcpp/types/Atom.h"
+#include "fhiclcpp/types/Sequence.h"
+
+#include "art/Framework/Core/ModuleMacros.h"
+#include "messagefacility/MessageLogger/MessageLogger.h"
+#include "MCDataProducts/inc/SimParticle.hh"
+#include "fhiclcpp/types/Sequence.h"
+#include "DataProducts/inc/PDGCode.hh"
+#include "MCDataProducts/inc/ProcessCode.hh"
+
+namespace mu2e {
+  class ParticleCodeFilter : public art::EDFilter {
+    public: 
+      using ParticleCodeConfig = fhicl::Sequence<fhicl::Tuple<int,int,int>>;
+
+      struct ModuleConfig {
+	using Name=fhicl::Name;
+	using Comment=fhicl::Comment;
+	fhicl::Atom<int> printLevel { Name("PrintLevel"), Comment ("Printout Level"), 0 };
+	fhicl::Atom<art::InputTag> simParticles { Name("SimParticles"), Comment("SimParticle collection") };
+	ParticleCodeConfig codeConfig { fhicl::Name("ParticleCodes") };
```

Comment:
As I read this, you need to put in a PDGCode and two ProcessCodes, all represented as integers.

I have a suggestion below for a future improvement.

We already have the infrastructure to enter the ProcessCodes as strings and convert them to the integer enum_type when they are read from the fcl file.  We sort of have it in HepPDT for PDGCodes but it's clunky.

.  On Sophie's PR she is putting in long lists of PDGCodes as integers:

    particles :   [-11,11,13,-13,2212,2112,211,22,212]

The above numbers appear in dozens of places fcl files in both Offline and Production.  

Does it make sense to change these to names instead of numbers?   It would make it less error prone and much more friendly to non-experts.